### PR TITLE
Nitiwiki next

### PR DESCRIPTION
**Pull request by Morriar**
Some fixes for `nitiwiki`.

Preparing for nitlanguage.org migration from ikiwiki to nitiwiki.

### CODE REVIEW

File: contrib/nitiwiki/src/wiki_html.nit [hunk 1]
```diff
@@ -218,8 +231,7 @@ redef class WikiArticle
 		while iter.is_ok do
 			var hl = iter.item
 			# parse title as markdown
-			var title = hl.title.md_to_html.to_s
-			title = title.substring(3, title.length - 8)
+			var title = wiki.inline_md(hl.title)
```
Comment (privat) [inline contrib/nitiwiki/src/wiki_html.nit]: htmlescape?

File: lib/markdown/decorators.nit [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+	redef fun add_paragraph(v, block) do v.emit_in block
+	redef fun add_headline(v, block) do v.emit_in block
+
+	redef fun add_code(v, block) do
+		v.add "<code>"
+		v.emit_in block
+		v.add "</code>"
+	end
+end
+
```
Comment (privat) [inline lib/markdown/decorators.nit]: trailing blank line

**Comment by privat** [comment]
+1

File: contrib/nitiwiki/src/wiki_html.nit (hunk 1 above)
Comment (Morriar) [inline contrib/nitiwiki/src/wiki_html.nit]: No, you want the decorated HTML

**Comment by Morriar** [comment]
Reroll without the blank line

File: contrib/nitiwiki/src/wiki_html.nit (hunk 1 above)
Comment (privat) [inline contrib/nitiwiki/src/wiki_html.nit]: Yes in fact, I want that.

File: contrib/nitiwiki/src/wiki_html.nit [hunk 3]
```diff
@@ -45,6 +46,18 @@ redef class Nitiwiki
 		sitemap.is_dirty = true
 		return sitemap
 	end
+
+	# Markdown processor used for inline element such as titles in TOC.
```
Comment (R4PaSs) [inline contrib/nitiwiki/src/wiki_html.nit]: :s/for/to

**Comment by R4PaSs** [comment]
:shipit:

File: lib/markdown/decorators.nit [hunk 4]
```diff
@@ -0,0 +1,17 @@
+import markdown
```
Comment (privat) [inline lib/markdown/decorators.nit]: license

**Comment by BlackMinou** [comment]
:shipit:

**Comment by Morriar** [comment]
Reroll with the licence

**Pull request merged**